Applications/Spreadsheet/CellType/Type.h

```diff
@@ -52,8 +52,13 @@ class CellType {
     virtual JS::Value js_value(Cell&, const CellTypeMetadata&) const = 0;
     virtual ~CellType() { }
 
+    const StringView& name() const { return m_name; }
+
 protected:
     CellType(const StringView& name);
+
+private:
+    StringView m_name;
```

Comment:
This feels sketchy. Who guarantees that the data inside this `StringView` doesn't disappear?

Reply:
The fact that all the cell types are meant to be constructed once, and are static (in `CellType/Type.cpp`)?
I _could_ make it allocate a String, but I don't think anyone would even _want_ to instantiate one of these

Reply:
Less sketch for a few more allocs :shrug: